src/components/pages/Create.vue

```diff
@@ -51,33 +60,41 @@ export default {
       const gh = new GitHub({
         token: this.state.token,
       });
+
       const options = {
         name: this.garment.title,
-        description: this.garment.title,
+        description: this.garment.description,
         homepage: 'http://example.com',
         auto_init: true,
       };
+
+      const garmentConfigOptions = {
+        title: 'github-for-fashion',
+        description: this.garment.description,
+        type: this.garment.type,
+        category: this.garment.category,
+        sizes: this.garment.sizes,
+        licence: this.garment.licence,
+        status: 'Created',
+      };
+
       const garmentConfig = 'garment-config.json';
+
       return gh.getUser().getProfile()
-      .then(function (profileResponse) {
-        console.log(profileResponse.data.login); // Success!
-        const username = profileResponse.data.login;
-        return gh.getUser(username).createRepo(options).then(function (createResponse) {
-          const remoteRepo = gh.getRepo(username, createResponse.data.name);
-          const obj = {
-            title: 'github-for-fashion',
-          };
-          return remoteRepo.writeFile('master', garmentConfig, JSON.stringify(obj), 'Garment project setup', function () {})
-          .then(function (repoResponse) {
-            console.log('success');
-            console.log(repoResponse);
+        .then(function (profileResponse) {
+          const user = profileResponse.data.login;
+          return gh.getUser(user).createRepo(options).then(function (createResponse) {
+            const repo = createResponse.data.name;
+            const remoteRepo = gh.getRepo(user, repo);
+            return remoteRepo.writeFile('master', garmentConfig, JSON.stringify(garmentConfigOptions), 'Garment project setup', function () {})
```

Comment:
Empty callback/object is necessary for the API call to work (represents the option parameter).